fix classes() for svg elements

jeffcunat opened the pull request:
Hello I had a problem using classes function on svg elements (same as #362).
This pull request should fix it as className returns a SVGAnimatedString object for svg elements instead of a simple string.
See notes in [(https://developer.mozilla.org/fr/docs/Web/API/Element/className)]

Commit:
fix classes for svg element

Commit:
fix eslint

Commit:
simplify fix

jeffcunat committed:
remove spaces

jeffcunat committed:
Update wrapper.js

eddyerburgh reviewed (changes requested):
Thanks for the PR

Can you add a unit test to check this please

Commit:
add unit test with svg element for classes

jeffcunat commented:
Hello Edd,
First of all, nice work with vue-test-utils, I use it every day now !
I have modified the unit test for classes to test svg elements. With dev branch it says : 
TypeError: this.element.className.split is not a function
With the PR everything's OK

eddyerburgh reviewed (approved):
Thanks for adding the test 😀
